### PR TITLE
Fix off-by-one indexing error in rotated-second-order cone code (#659)

### PR DESCRIPTION
**Pull request by joehuchette**
This needs rSOC support in MPB (https://github.com/JuliaOpt/MathProgBase.jl/blob/2f85f363cbb4ea24997706f9d2c04f77b7fb808e/src/SolverInterface/lpqp_to_conic.jl#L38) to actually work, I think.

### CODE REVIEW

**Comment by joehuchette** [comment]
ref #659

File: test/qcqpmodel.jl [hunk 1]
```diff
@@ -355,3 +355,23 @@ context("With solver $(typeof(solver))") do
     @fact solve(modQ) --> :Optimal
     @fact getObjectiveValue(modQ) --> roughly(-0.75, 1e-6)
 end; end; end
+
+facts("[qcqpmodel] Rotated second-order cones") do
+for solver in soc_solvers
```
Comment (mlubin) [inline test/qcqpmodel.jl]: ``soc_solver`` won't work since ECOS and SCS don't support rotated second-order cones

**Comment by mlubin** [comment]
The ``1.0`` coefficients should be changed here https://github.com/JuliaOpt/JuMP.jl/blob/4513be6fdc300ed69273d1edff549cfd001c3317/src/solvers.jl#L862. I think it should be ``append!(V, [-0.5; -1.0; ones(n-2)])``

**Comment by joehuchette** [comment]
Yeah, I did
```
append!(V, [-1/sqrt(2); -1/sqrt(2); ones(n-2)])
```

**Comment by mlubin** [comment]
At least the nightlies are working on windows again.

**Comment by joehuchette** [comment]
Should probably backport

**Comment by mlubin** [comment]
Sure, but probably not worth a new release yet

**Comment by joehuchette** [comment]
d4c0cc1eaaa9263d16b611cb656ff3c7808ef2be
68373eac9c54ac2e2eb30dd53aa814f10b8861b2
f77dcac8ca046028f5cba8240f823be665b2e257

**Pull request merged**